<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="ViewFieldGL" name="openGLWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="filenameLabel">
          <property name="text">
           <string>Filename:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="filePathLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>no file opened</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="newFileButton">
          <property name="text">
           <string>new</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="screenshotButton">
            <property name="text">
             <string>Screenshot</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="gifButton">
            <property name="text">
             <string>GIF</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="7" column="0" colspan="3">
       <widget class="QSlider" name="scaleSlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
        <property name="sliderPosition">
         <number>25</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="xOffsetLabel">
        <property name="font">
         <font>
          <family>Noto Sans Linear A</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="3">
       <widget class="QComboBox" name="projectionTypeCmbBox">
        <property name="font">
         <font>
          <family>Noto Sans Linear A</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Central</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Parallel</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QDoubleSpinBox" name="xRotSpBox">
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QDoubleSpinBox" name="zOffstSpBox">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>-999.990000000000009</double>
        </property>
        <property name="maximum">
         <double>999.990000000000009</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="yOffsetLabel">
        <property name="font">
         <font>
          <family>Noto Sans Linear A</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="0" colspan="3">
       <widget class="QLabel" name="projTypeLabel">
        <property name="font">
         <font>
          <family>Noto Sans Linear A</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Projection type:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="zOffsetLabel">
        <property name="font">
         <font>
          <family>Noto Sans Linear A</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Z:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="zRotSpBox">
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QLabel" name="scaleLabel">
        <property name="font">
         <font>
          <family>Noto Sans Linear A</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Scale</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QLabel" name="offsetLabel">
        <property name="font">
         <font>
          <family>Noto Sans Linear A</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Offset</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="angleRotLabel">
        <property name="font">
         <font>
          <family>Noto Sans Linear A</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Angle rotation</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QDoubleSpinBox" name="xOffstSpBox">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>-999.990000000000009</double>
        </property>
        <property name="maximum">
         <double>999.990000000000009</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="zRotLabel">
        <property name="font">
         <font>
          <family>Noto Sans Linear A</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Z:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="yRotSpBox">
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="14" column="0" colspan="3">
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Noto Sans Linear A</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="vrtxTab">
         <attribute name="title">
          <string>Vertexes</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="vrtxShapeLabel">
              <property name="font">
               <font>
                <family>Noto Sans Linear A</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Shape:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="vrtxShapeCmbBox">
              <property name="font">
               <font>
                <family>Noto Sans Linear A</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <item>
               <property name="text">
                <string>Circle</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Square</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>None</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="vrtxSizeLabel">
            <property name="font">
             <font>
              <family>Noto Sans Linear A</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Size:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="vrtxSizeSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="vrtxColorLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Noto Sans Linear A</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Colour:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="vrtxColorButton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="edgeTab">
         <attribute name="title">
          <string>Edges</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="edgesTypeLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Noto Sans Linear A</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Type:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="edgesTypeCmbBox">
              <property name="font">
               <font>
                <family>Noto Sans Linear A</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <item>
               <property name="text">
                <string>Solid</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Dotted</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="edgesWidthLabel">
            <property name="font">
             <font>
              <family>Noto Sans Linear A</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Width:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="edgesWidthSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="edgesColorLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Noto Sans Linear A</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Colour:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="edgesColorButton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="xRotLabel">
        <property name="font">
         <font>
          <family>Noto Sans Linear A</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="yRotLabel">
        <property name="font">
         <font>
          <family>Noto Sans Linear A</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="yOffstSpBox">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>-999.990000000000009</double>
        </property>
        <property name="maximum">
         <double>999.990000000000009</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="bgClrLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Noto Sans Linear A</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Background colour:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="bgClrButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ViewFieldGL</class>
   <extends>QOpenGLWidget</extends>
   <header>viewfieldgl.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
